Firestore/core/src/firebase/firestore/remote/grpc_root_certificate_finder_apple.mm

```diff
@@ -0,0 +1,50 @@
+/*
```

Comment:
You'll also need to add these new files to the cmake build. But I seem to recall you mentioning that you wanted to tackle cmake in a followup? If so; that's fine. No action required.